tests/visualization_tests/test_pareto_front.py

```diff
@@ -1,451 +1,230 @@
-@pytest.mark.parametrize("axis_order", [None, [0, 1], [1, 0]])
-@pytest.mark.parametrize("targets", [None, lambda t: (t.values[0], t.values[1])])
-def test_plot_pareto_front_2d(
-    include_dominated_trials: bool,
-    use_constraints_func: bool,
-    axis_order: Optional[List[int]],
-    targets: Optional[Callable[[FrozenTrial], Sequence[float]]],
-) -> None:
-    if axis_order is not None and targets is not None:
-        pytest.skip("skip using both axis_order and targets")
+def create_study_2d() -> Study:
 
-    # Test with no trial.
     study = optuna.create_study(directions=["minimize", "minimize"])
-    figure = plot_pareto_front(
-        study=study,
-        include_dominated_trials=include_dominated_trials,
-        axis_order=axis_order,
-    )
-    assert len(figure.data) == 2
-    assert (figure.data[1]["x"] + figure.data[0]["x"]) == ()
-    assert (figure.data[1]["y"] + figure.data[0]["y"]) == ()
 
-    # Test with four trials.
     study.enqueue_trial({"x": 1, "y": 2})
     study.enqueue_trial({"x": 1, "y": 1})
     study.enqueue_trial({"x": 0, "y": 2})
     study.enqueue_trial({"x": 1, "y": 0})
     study.optimize(lambda t: [t.suggest_int("x", 0, 2), t.suggest_int("y", 0, 2)], n_trials=4)
+    return study
 
-    constraints_func: Optional[Callable[[FrozenTrial], Sequence[float]]]
-    if use_constraints_func:
-        # (x, y) = (1, 0) is infeasible; others are feasible.
-        def constraints_func(t: FrozenTrial) -> Sequence[float]:
-            return [1.0] if t.params["x"] == 1 and t.params["y"] == 0 else [-1.0]
 
-    else:
-        constraints_func = None
+def create_study_3d() -> Study:
 
-    figure = plot_pareto_front(
-        study=study,
-        include_dominated_trials=include_dominated_trials,
-        axis_order=axis_order,
-        constraints_func=constraints_func,
-        targets=targets,
-    )
-    actual_axis_order = axis_order or [0, 1]
-    data: List[Tuple[int, ...]]
-    if use_constraints_func:
-        assert len(figure.data) == 3
-        if include_dominated_trials:
-            # The enqueue order of trial is: infeasible, feasible non-best, then feasible best.
-            data = [(1, 0, 1, 1), (1, 2, 2, 0)]
-        else:
-            # The enqueue order of trial is: infeasible, feasible.
-            data = [(1, 0, 1), (1, 2, 0)]
-    else:
-        assert len(figure.data) == 2
-        if include_dominated_trials:
-            # The last elements come from dominated trial that is enqueued firstly.
-            data = [(0, 1, 1, 1), (2, 0, 2, 1)]
-        else:
-            data = [(0, 1), (2, 0)]
-
-    _check_data(figure, "x", data[actual_axis_order[0]])
-    _check_data(figure, "y", data[actual_axis_order[1]])
+    study = optuna.create_study(directions=["minimize", "minimize"])
 
-    titles = ["Objective {}".format(i) for i in range(2)]
-    assert figure.layout.xaxis.title.text == titles[actual_axis_order[0]]
-    assert figure.layout.yaxis.title.text == titles[actual_axis_order[1]]
```

Comment:
Split into `test_get_pareto_front_info_unconstrained` and `test_get_pareto_front_info_constrained`.